lnst/RecipeCommon/Perf/Measurements/BaseFlowMeasurement.py

```diff
@@ -297,8 +302,16 @@ def _report_flow_results(cls, recipe, flow_results):
                         cpu_deviation=receiver_cpu.std_deviation,
                         cpu_unit=receiver_cpu.unit))
 
-        #TODO add flow description
-        recipe.add_result(True, "\n".join(desc), data = dict(
+        recipe_result = True
+        flows = {"Generator": generator, "Generator process": generator_cpu,
```

Comment:
nitpick here... these are **metrics** or **measurement results**, not flows. all of these metrics are related to the same flow measurement.

Reply:
Thanks, i just changed that.